### PR TITLE
Fix typo where request.method is request.path

### PR DESCRIPTION
**Pull request by hubt**
There is a similar problem in the docs.

### CODE REVIEW

**Comment by ldemailly** [comment]
/ok-to-test
I have some deja vu somehow on this change, cc @douglas-reid

File: samples/apps/bookinfo/mixer-rule-additional-telemetry.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ rules:
           source: source.labels["app"] | "unknown"
           target: target.service | "unknown"
           service: target.labels["app"] | "unknown"
-          method: request.path | "unknown"
+          method: request.method | "unknown"
```
Comment (douglas-reid) [inline samples/apps/bookinfo/mixer-rule-additional-telemetry.yaml]: this was intentional -- method is just a bit overloaded. (it was meant to get at the API method of the service `/ratings` v. the http method `GET`). But I have no strong feelings on the need for it either way.

**Comment by sebastienvas** [comment]
Should we close this ?

**Comment by ldemailly** [comment]
closing but reopen if I'm reading this wrong

**Pull request closed without merging**